remoulade/state/backend.py

```diff
@@ -0,0 +1,62 @@
+from typing import Iterable
+
+from ..encoder import Encoder
+
+
+class State:
```

Comment:
I don't think we need this. as long as there is no usefull method to State we can work with a dict.